packages/applications/ssr/src/app/laureats/[identifiant]/puissance/changement/[date]/page.tsx

```diff
@@ -0,0 +1,104 @@
+import { Puissance } from '@potentiel-domain/laureat';
+import { mapToPlainObject } from '@potentiel-domain/core';
+import { IdentifiantProjet } from '@potentiel-domain/common';
+import { Role } from '@potentiel-domain/utilisateur';
+
+import {
+  ChangementPuissanceActions,
+  DétailsPuissancePage,
+} from '@/components/pages/puissance/changement/détails/DétailsPuissance.page';
+import { decodeParameter } from '@/utils/decodeParameter';
+import { PageWithErrorHandling } from '@/utils/PageWithErrorHandling';
+import { withUtilisateur } from '@/utils/withUtilisateur';
+
+export const metadata: Metadata = {
+  title: 'Détail de la puissance du projet - Potentiel',
+  description: 'Détail de la puissance du projet',
+};
+
+type PageProps = {
+  params: {
+    identifiant: string;
+    date: string;
+  };
+};
+
+export default async function Page({ params: { identifiant, date } }: PageProps) {
+  return PageWithErrorHandling(async () =>
+    withUtilisateur(async (utilisateur) => {
+      const identifiantProjet = IdentifiantProjet.convertirEnValueType(
+        decodeParameter(identifiant),
+      );
+
+      const demandéLe = decodeParameter(date);
+
+      const changement = await mediator.send<Puissance.ConsulterChangementPuissanceQuery>({
+        type: 'Lauréat.Puissance.Query.ConsulterChangementPuissance',
+        data: {
+          identifiantProjet: identifiantProjet.formatter(),
+          demandéLe,
+        },
+      });
+
+      if (Option.isNone(changement)) {
+        return notFound();
+      }
+
+      const puissance = await mediator.send<Puissance.ConsulterPuissanceQuery>({
+        type: 'Lauréat.Puissance.Query.ConsulterPuissance',
+        data: {
+          identifiantProjet: identifiantProjet.formatter(),
+        },
+      });
+
+      if (Option.isNone(puissance)) {
+        return notFound();
+      }
+
+      return (
+        <DétailsPuissancePage
+          identifiantProjet={mapToPlainObject(identifiantProjet)}
+          demande={mapToPlainObject(changement.demande)}
+          actions={mapToActions(
+            changement.demande.statut,
+            utilisateur.role,
+            !changement.demande.isInformationEnregistrée
+              ? changement.demande.autoritéCompétente
+              : undefined,
+          )}
+          demandeEnCoursDate={
+            puissance.dateDemandeEnCours ? puissance.dateDemandeEnCours.formatter() : undefined
+          }
+        />
+      );
+    }),
+  );
+}
+
+const mapToActions = (
+  statut: Puissance.StatutChangementPuissance.ValueType,
+  rôle: Role.ValueType,
+  autoritéCompétente?: Puissance.RatioChangementPuissance.AutoritéCompétente,
+): Array<ChangementPuissanceActions> => {
+  const actions: Array<ChangementPuissanceActions> = [];
+
+  if (statut.estDemandé()) {
+    const estAutoritéCompétente =
+      autoritéCompétente === rôle.nom ||
+      (autoritéCompétente === 'admin' && rôle.nom === 'dgec-validateur');
```

Comment:
Ici faut aussi inclure les admins :
```suggestion
      (autoritéCompétente === 'admin' && (rôle.nom === 'dgec-validateur' || rôle.nom === 'admin'));
```